rosdep/python.yaml

```diff
@@ -3055,13 +3055,13 @@ python-omniorb:
 python-open3d-pip:
   debian:
     pip:
-      packages: [open3d-python]
+      packages: [open3d-python, open3d]
   fedora:
     pip:
-      packages: [open3d-python]
+      packages: [open3d-python, open3d]
   ubuntu:
     pip:
-      packages: [open3d-python]
+      packages: [open3d-python, open3d]
```

Comment:
It looks like `open3d` is python3 only, while `open3d-python` was python2 compatible.
i.e.: `python-open3d-pip` shouldn't be modified, only `python3-open3d-pip`